src/device-registry/models/Event.js

```diff
@@ -563,10 +563,21 @@ eventSchema.statics = {
 
       logObject("the query for this request", search);
       if (!recent || recent === "yes") {
-        let data = await this.aggregate()
+        /**
+         * look up the health tips and attach them to
+         * measurements as you return each one of them
+         * what is the direct relationship between measurement and pm2.5?
+         */
+        const data = await this.aggregate()
           .unwind("values")
           .match(search)
           .replaceRoot("values")
+          .lookup({
```

Comment:
where the look up for health tips, using the pm2.5 value of measurements is supposed to happen from here. @BenjaminSsempala